src/Robo/Commands/Sync/SyncCommand.php

```diff
@@ -37,18 +37,24 @@ public function allSites() {
    * Copies remote db to local db, re-imports config, and executes db updates
    * for each multisite.
    *
+   * @param array $options
+   *   Array of CLI options.
+   *
    * @command drupal:sync:default:site
    * @aliases ds drupal:sync drupal:sync:default sync sync:refresh
    * @executeInVm
    */
-  public function sync($options = [
+  public function sync(array $options = [
     'sync-files' => FALSE,
+    'sync-private-files' => FALSE,
```

Comment:
Is this a public vs private? Can't tell now. If so it would be good to have at least an alias sync:public-files for sync:files.

Reply:
Good point, I'll add an alias and we can make the alias canonical in BLT 11.x

Reply:
https://github.com/acquia/blt/issues/3860